main.go

```diff
@@ -84,8 +86,10 @@ func main() {
 
 	var enableHTTP2 = false
 
+	var labelSelectorFlag string
 	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
 	flag.StringVar(&probeAddr, "health-probe-bind-address", ":8081", "The address the probe endpoint binds to.")
+	flag.StringVar(&labelSelectorFlag, "label-selector", env.StringFromEnv(argocdcommon.ArgoCDLabelSelectorKey, argocdcommon.ArgoCDDefaultLabelSelector), "The label selector is used to map to a subset of ArgoCD instances to reconcile")
```

Comment:
Can you replace `env.StringFromEnv(argocdcommon.ArgoCDLabelSelectorKey, argocdcommon.ArgoCDDefaultLabelSelector)` and remove the dependency of `"github.com/argoproj/argo-cd/v2/util/env"`